theforeman.org/pipelines/test/smart-proxy-stable.groovy

```diff
@@ -0,0 +1,113 @@
+git_url = 'https://github.com/theforeman/smart-proxy'
+rubocop_ruby = '2.7'
+
+pipeline {
+    agent none
+    options {
+        timeout(time: 1, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+    environment {
+        BUNDLE_JOBS = 4
+        BUNDLE_RETRY = 3
+    }
+
+    stages {
+        stage('Rubocop') {
+            agent any
+            environment {
+                BUNDLE_WITHOUT = 'bmc:development:dhcp_isc_inotify:dhcp_isc_kqueue:journald:krb5:libvirt:puppetca_token_whitelisting:realm_freeipa:windows'
+            }
+
+            stages {
+                stage('Setup Git Repos') {
+                    steps {
+                        git(url: git_url, branch: git_branch)
```

Comment:
This is the only (real) difference to the PR testing pipeline. I'm wondering if we should externalize the git checkout step to a function.

Reply:
I now applied this in the first commit.